src/Main/Main.js

```diff
@@ -24,6 +24,7 @@ const INITIAL_RESULT_COUNT = 30;
 const RESULT_COUNT_INCREMENT = 30;
 const filterConfig = Filters();
 const searchableIndexes = SearchableIndexes;
+const EMPTY_DROPDOWN_ITEM = { label: '', value: '' };
```

Comment:
Maybe it's better to keep this const in `Utils`, cause we are going to use it in different parts of the app.

Reply:
Yes, sure, I moved.